modules/dnn/test/test_layers.cpp

```diff
@@ -1007,11 +1016,14 @@ TEST_P(Layer_Test_DWconv_Prelu, Accuracy)
     LayerParams lp;
     lp.name = "dwconv";
     lp.type = "Convolution";
-    lp.set("kernel_size", 3);
+    std::vector<int> kernel_size(2, 3);
+    lp.set("kernel_size", DictValue::arrayInt<int*>(kernel_size.data(), kernel_size.size()));
     lp.set("num_output", num_output);
-    lp.set("pad", 0);
+    std::vector<int> pad(2, 0);
+    lp.set("pad", DictValue::arrayInt<int*>(pad.data(), pad.size()));
     lp.set("group", group);
-    lp.set("stride", 1);
+    std::vector<int> stride(2, 1);
+    lp.set("stride", DictValue::arrayInt<int*>(stride.data(), stride.size()));
```

Comment:
PR going to support Conv1D for CUDA but we already integrated Conv1D and it works with single value paddings, strides and kernel_size. So why for this backend we need to change common part?

Reply:
> PR going to support Conv1D for CUDA but we already integrated Conv1D and it works with single value paddings, strides and kernel_size. So why for this backend we need to change common part?

This changes are needed for CUDA, but I think we should have the similar CUDA-independent logic for ```master``` and ```3.4``` branches

Reply:
How changes in tests are required to support specific backend?
Necessary logic should be handled by backend implementation itself.

Reply:
> changes in tests
I removed changes in tests. I just want to preserve the same code and logic for backend-independent parts in ```master``` and ```3.4```